### ISSUE TITLE
Sequencer Swing

### ISSUE THREAD

**Issue by UltraBlackLinux** [rolfdegen/Jeannie-Open-source-Synthesizer#19]
Hey there,
I just though of another feature that I would really like to use: A swing rhythm for the sequencer. Many sequencers have that and I feel like this fits Jeannie perfectly. 
Thanks!

**Comment by rolfdegen** [comment]
Yes, that's a good idea and will come with the next sequencer update.
